module.js

```diff
@@ -118,9 +118,10 @@ const init = async (input) => {
   const imports = getImports();
 
   if (
-    typeof input === "string" ||
+    // cannot fetch from file system
+    (typeof input === "string" && input.indexOf("file:") !== 0) ||
```

Comment:
`fetch` doesn't work if the url is a file, since there's no server running.  Adding these checks here doesn't really fix anything, but it does result in an error message of "TypeError: WebAssembly.instantiate(): Argument 0 must be a buffer source" instead of "TypeError: fetch failed".  Not totally sure if that's better, but seems more useful.